public/app/features/alerting/unified/components/rules/RulesFilter.tsx

```diff
@@ -232,7 +232,7 @@ const RulesFilter = ({ onFilterCleared = () => undefined }: RulesFilerProps) =>
                     <Stack gap={0.5}>
                       <span>Search</span>
                       <HoverCard content={<SearchQueryHelp />}>
-                        <Icon name="info-circle" size="sm" tabIndex={0} />
+                        <Icon name="info-circle" size="sm" tabIndex={0} title="Search help" />
                       </HoverCard>
```

Comment:
The popup from the `HoverCard` is not keyboard accessible. It should be changed to the `ToggleTip` component.

Reply:
Would it not be better to make these changes in a separate PR and just leave this one only to improve the documentation and default the Icon to `aria-hidden`? 🤔